cli/ceph-disk

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+/usr/bin/docker run --rm --privileged=true -v /etc/ceph:/etc/ceph -v /dev:/dev ceph/base ceph-disk "$@"
```

Comment:
I think you should also expose `var/lib/ceph/bootstrap-osd` so `ceph-disk` can add new OSDs using the bootstrap key.
I did some (unsuccessful) experiment trying to get `ceph-disk activate` I'll be happy to hear if you got it working :).